font/src/directwrite/mod.rs

```diff
@@ -51,9 +189,12 @@ impl crate::Rasterize for DirectWriteRasterizer {
         let line_height = f64::from(ascent - descent + line_gap);
 
         // We assume that all monospace characters have the same width
-        // Because of this we take '!', the first drawable character, for measurements
-        let glyph_metrics = font.get_design_glyph_metrics(&[33], false);
-        let hmetrics = glyph_metrics.first().ok_or(Error::MissingGlyph('!'))?;
+        // Because of this we take '!', the first drawable character, for horizontal metrics
```

Comment:
```
        // Since all monospace characters have the same width, we use `!` for horizontal metrics
```

I think this works a bit better, even though it has only slightly been changed in this PR. That way we don't have two sentences that aren't properly delimited with punctuation but still separate sentences.